sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/Samples/Sample01_HelloWorld.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+#region Snippet:ContainerRegistry_Tests_Samples_Namespaces
+using Azure.Containers.ContainerRegistry;
+#endregion Snippet:ContainerRegistry_Tests_Samples_Namespaces
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.Containers.ContainerRegistry.Tests.Samples
+{
+    public partial class HelloWorld : SamplesBase<ContainerRegistryTestEnvironment>
+    {
+        [Test]
+        [SyncOnly]
+        public void CreateClient()
+        {
+            Environment.SetEnvironmentVariable("REGISTRY_ENDPOINT", TestEnvironment.Endpoint);
+
+            #region Snippet:ContainerRegistry_Tests_Samples_CreateClient
+            // Get the service endpoint from the environment
+            Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
```

Comment:
Is this just to avoid having to include TestEnvironment.Endpoint in the sample?
You can use the //@@ syntax to include content that will be rendered in the markdown, e.g. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample01_HelloWorld.cs#L24

Reply:
Copying Ted 😊: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/tests/Samples/Sample01_HelloWorld.cs#L36